statsmodels/stats/tests/test_nonparametric.py

```diff
@@ -493,3 +495,277 @@ def test_rank_compare_vectorized():
         tost_i = res_i.tost_prob_superior(0.4, 0.6)
         assert_allclose(tost.statistic[i], tost_i.statistic, rtol=1e-14)
         assert_allclose(tost.pvalue[i], tost_i.pvalue, rtol=1e-14)
+
+@pytest.mark.parametrize(
+    "x1, x2, expected_holder",
+    [
```

Comment:
To improve readability, I usually define the list of cases in a separate list.

The advantage is also that sometimes I use then the list in several parameterized unit test, e.g. if those are added later

Reply:
Just to make sure I handle this correctly. I'm thinking of doing the following: 

```python
@pytest.fixture(scope="module")
def two_sample_continuous():
    """
    This fixture generates two samples of
    continuous data without ties.
    """
    x1 = np.array([1.1, 2.2, 3.3, 4.4, 5.5])
    x2 = np.array([6.6, 7.7, 8.8, 9.9, 10.1])
    return x1, x2

@pytest.fixture(scope="module")
def two_sample_ordinal():
    """
    This fixture generates two samples of
    ordinal data with ties.
    """
    x1 = np.array([1, 1, 2, 2, 3])
    x2 = np.array([4, 5, 6, 7, 8])
    return x1, x2
   
# These can be used in parametrized tests like

```python
@pytest.mark.parametrize(
    "data_samples, expected_holder",
    [
        # No ties continuous data
        (
            "two_sample_continuous",
             ...,
        ),
        # Ties ordinal data
        (
            "two_sample_ordinal",
            ...,
        ),
    ],
)
def test_compute_rank_placements(request, data_samples, expected_holder):
    # Get the fixture values
    x1, x2 = request.getfixturevalue(data_samples)
```

Do you think this does the trick? Or perhaps a simpler set up is to create these as a lists of two tuples(x1, x2) at the top of the module directly without making them fixtures.

Reply:
It's simpler just to use a list
(I never used fixtures. I never found them necessary for this.)

for example
https://github.com/statsmodels/statsmodels/blob/main/statsmodels/stats/tests/test_rates_poisson.py#L108
and many other cases in that module

and two lists of cases can be concatenated with just `+` in parametrize call